CollAction/Controllers/ProjectsController.cs

```diff
@@ -128,13 +129,15 @@ public ViewResult StartInfo()
                 return View(model);
             }
 
+            // TODO: check incoming HTML for invalid tags
```

Comment:
I think that's done here, looking at the code below. Bit confusing to leave "done" stuff as TODO comment.